library/src/main/java/com/datatorrent/lib/parser/Parser.java

```diff
@@ -61,25 +70,48 @@ public void setup(PortContext context)
     }
   };
 
-  @OutputPortFieldAnnotation(optional = true)
   public transient DefaultOutputPort<INPUT> err = new DefaultOutputPort<INPUT>();
-
+  public transient DefaultOutputPort<INPUT> validatedOutput = new DefaultOutputPort<INPUT>();
```

Comment:
Do we need this ? We had a discussion on eng regarding whether this port would be useful and the opinion was this wont be used.